articles/openshift/oshift-how-apply-security-best-practices.md

```diff
@@ -0,0 +1,70 @@
+---
+title: How to apply security best practices in OpenShift
+description: Apply security best practices in OpenShift
+services: openshift
+author: Mudasar Hussain
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Apply security best practices in OpenShift
+toc_fullpath: How To/apply-security-best-practices.md
+toc_mdlink: oshift-how-apply-security-best-practices.md
+---
+
+# How to apply security best practices in OpenShift
+
+## Overview
+
+This knowledge centre article sets out some of the best practices which can be adhered to in order to minimise the attack vector to a UKCloud OpenShift cluster and its workloads. As the nature of running container-based workloads reduces the threat to your underlying infrastructure, it is not completely eliminated, as such great care and attention must be taken when designing, developing and deploying your container-based workloads to minimise the risks to your container-based deployments in UKCloud OpenShift. Unfortunately, as is the nature of technology any cloud-based service is vulnerable to exploits from both hackers and insiders just the same as traditional environments. There are risks of data stealing, crypto mining, ransomware and service disruption to name a few.
+
+### Intended audience
+
+This article is at everybody who is using UKCloud OpenShift. It will highlight just some of the best practices to keep OpenShift workloads secure.
+
+### Security best practices for UKCloud OpenShift
+
+- Role Based Access Control - Within OpenShift users are granted various levels of access which can be cluster-wide or limited to a particular project within OpenShift. Both the users and groups can be associated with multiple roles at the same time. Users with the cluster-admin default cluster role bound cluster-wide can perform any action on any resource. Users with the admin default cluster role bound locally can manage roles and bindings in that project. Please see a pre-defined list of cluster roles and local roles [here] (https://docs.openshift.com/container-platform/3.11/architecture/additional_concepts/authorization.html#roles). It is good practice to follow the 'Principle of least Access' with OpenShift access. Customers should follow the following guide to create [new OpenShift users](https://docs.ukcloud.com/articles/openshift/oshift-how-create-users.html). This guide also explains how to enable two-factor authentication (2FA) when logging in to OpenShift. Users are also capable of changing their passwords regularly from the UKCloud portal.
```

Comment:
Is there a web page we can point to that provides more information about the Principle of Least Access?